.circleci/config.yml

```diff
@@ -59,8 +70,7 @@ shared: &shared
     - save_cache:
         paths:
           - ./vendor/bundle
-          - ./ccache
-        key: v1-dependencies-{{ checksum "Gemfile.lock" }}
```

Comment:
`Gemfile.lock` was not included in this repository.
So `restore_cache` phase will be failed.